src/PrintLoopNest.h

```diff
@@ -17,7 +17,7 @@ class Function;
 /** Emit some simple pseudocode that shows the structure of the loop
  * nest specified by this pipeline's schedule, and the schedules of
  * the functions it uses. */
-std::string print_loop_nest(const std::vector<Function> &outputs);
+std::string print_loop_nest(std::vector<Function> outputs);
```

Comment:
This seems to be exposing an aspect of the internal implementation of the function solely to save one line of code in the internals. I'd leave it a const ref, rename the formal parameter and declare a local outputs vector and use that in the tie.